src/resources/namespaced/organization-organization-venues.ts

```diff
@@ -0,0 +1,55 @@
+import { createRequestMethod } from "../../interfaces/server/request-method.interface";
+import ResourceClass from "../../classes/abstracts/resource.class";
+import { IndexInterface } from "../../interfaces/resources/structures/index.interface";
+import { OrganizationInterface } from "../../interfaces/resources/organization.interface";
+
+/**
+ * @endpoint organizations.organizations_venues
+ * @url /organizations/{id}/organizations_venues
+ */
+class OrganizationOrganizationVenues extends ResourceClass {
```

Comment:
This should be plural
```suggestion
class OrganizationOrganizationsVenues extends ResourceClass {
```